### PR TITLE
Fix uncommon evolution methods

### PR DESCRIPTION
**Pull request by KrisXV**

### CODE REVIEW

File: data/pokedex.ts [hunk 1]
```diff
@@ -11650,6 +11654,7 @@ export const BattlePokedex: {[speciesid: string]: SpeciesData} = {
 		color: "Blue",
 		prevo: "Inkay",
 		evoLevel: 30,
+		evoCondition: "with an console turned upside-down",
```
Comment (urkerab) [inline data/pokedex.ts:11657]:
```suggestion
		evoCondition: "with the console turned upside-down",
```

Comment (KrisXV) [reply data/pokedex.ts:11657]: I don’t like the wording of “the”. I’ll fix it to “a” when I’m on a computer.

Comment (Zarel) [reply data/pokedex.ts:11657]: "the" is definitely better English.

**Comment by Zarel** [comment]
(Specifically, "the console" means "a specific console, you know which one I'm talking about, the one you're trying to evolve the Inkay on", while "a console" means "any of multiple consoles will work, it doesn't matter which one". So using "a console" here is incorrect English.)

**Pull request merged**